### ISSUE TITLE
Testing branch giving errors, does not start gui

### ISSUE THREAD

**Issue by aatz1982** [aclist/dztui#97]
Hi, just FYI the testing branch is currently not running for me. The current stable branch (4.1.1) is still running fine with the same settings. The testing version is version=5.0.0.rc-30 (same issue with rc-28 & rc-29, not tried any others recently). All of the dependencies I know of are up to date (curl, jq, zenity, steam, wmctrl, xdotool). The system is running Debian 12.

**Config file:**

#Path to DayZ installation
steam_path="/home/r/.steam/debian-installation"

#Battlemetrics API key
api_key="REDACTED"

#Favorited server IP:PORT array
ip_list=(

)

#Favorite server to fast-connect to (limit one)
fav_server=""

#Favorite server label (human readable)
fav_label=""

#Custom player name (optional, required by some servers)
name="Player1"

#Set to 1 to perform dry-run and print launch options
debug=""

#Toggle stable/testing branch
branch="testing"

#Last seen news item
seen_news="54946b41074f9899734e3e10cb95aea5"

#Steam API key
steam_api="REDACTED"

#Auto-install mods
auto_install="0"

#Automod staging directory
staging_dir="/tmp"

#Path to default Steam client
default_steam_path="/home/r/.steam/steam"

#Preferred Steam launch command (for Flatpak support)
preferred_client="steam"

#DZGUI source path
src_path="/home/r/.local/share/dzgui/dzgui.sh"

**Console output:**

r@debian:~/.local/share/dzgui$ ./dzgui.sh
Initializing setup...
All OK. Kicking off UI...
Traceback (most recent call last):
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 1655, in <module>
    main()
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 1651, in main
    App()
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 1483, in __init__
    self.win = OuterWindow(is_steam_deck)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 449, in __init__
    self.grid = Grid(is_steam_deck)
                ^^^^^^^^^^^^^^^^^^^
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 1400, in __init__
    self.right_panel = RightPanel(is_steam_deck)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 484, in __init__
    self.filters_vbox = FilterPanel()
                        ^^^^^^^^^^^^^
  File "/home/r/.local/share/dzgui/helpers/ui.py", line 1530, in __init__
    if query_config(None, "debug")[0] == '1':
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
Terminated

**Sys:**

r@debian:~/.local/share/dzgui$ uname -v
#1 SMP PREEMPT_DYNAMIC Debian 6.1.69-1 (2023-12-30)

Hope that's of use to you, thanks for your hard work!

**Comment by aclist** [comment]
Thanks for the heads up. I had fixed this locally but did not push the commit yet. It is updated now on rc-31.

How are you liking the beta version so far?

**Comment by aclist** [comment]
Resolved in 5.0.0
